[front] - fix(analytics_queue): store MCP server configuration id

JulesBelveze opened the pull request:
## Description

This PR changes `mcp_server_configuration_id` from `config.sId` to string using `config.id.toString()` ensuring stable configuration pointer.

## Tests

Locally

## Risk

Low - behind FF

## Deploy Plan

Deploy front

JulesBelveze committed:
[analytics_queue] - fix: correct property type for mcp_server_configuration_id

 - Change mcp_server_configuration_id from config.sId to string using config.id.toString() ensuring consistent data types in documents

aubin-tchoi reviewed (approved):
LGTM, but don't we want the mapping to be an `integer` instead?

JulesBelveze committed:
[analytics/indices] - fix: change mcp_server_configuration_id type to integer

 - Corrected the type of mcp_server_configuration_id from keyword to integer for proper numerical sorting and aggregations
